modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs

```diff
@@ -104,7 +104,7 @@ public static void OpenFile(string slnPath, string scriptPath, int line)
             if (line >= 0)
             {
                 args.Add("--line");
-                args.Add(line.ToString());
+                args.Add((line+1).ToString()); // https://github.com/JetBrains/godot-support/issues/61
```

Comment:
```suggestion
                args.Add((line + 1).ToString()); // https://github.com/JetBrains/godot-support/issues/61
```

Per the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html), spaces around operators.

Reply:
Thanks for noticing. Fixed